### PR TITLE
[Deep Learning commands] Update solutionsLib.py

### PR DESCRIPTION
**Pull request by surabhijain123**
Issue: https://github.com/Esri/mdcs-py/issues/127

1. Adding Deep Learning Commands in MDCS
- CPUDL: ClassifyPixelsUsingDeepLearning
- DOUDL: DetectObjectsUsingDeepLearning
- COUDL: ClassifyObjectsUsingDeepLearning
- EFUAIM: Extract Features Using AI Models

2. Fix in Set Environment in case input is '#'

### CODE REVIEW

**Comment by surabhijain123** [comment]
@Chamlika Please review this

**Comment by Chamlika** [comment]
@abhiataero This was approved earlier.

**Comment by Chamlika** [comment]
@surabhijain123 The changes that were added earlier aren't reflected here in the source here. It's possible the PR isn't attached to this particular issue #, can you check, thanks!

File: scripts/solutionsLib.py [hunk 1]
```diff
@@ -1370,6 +1370,62 @@ def executeCommand(self, com, index=0):
                 'arcpy.TransferFiles_management',
                 index
             )
+
```
Comment (Chamlika) [inline scripts/solutionsLib.py:1373]: Please remove empty line, thanks!

File: scripts/solutionsLib.py [hunk 2]
```diff
@@ -1370,6 +1370,62 @@ def executeCommand(self, com, index=0):
...
+            index
+        )
+
+        elif (com == 'COUDL'):
+            self.m_log.Message("\t{}:{}".format(self.commands[com]['desc'], self.m_base.m_mdName), self.m_log.const_general_text)
+            processKey = 'classifyobjectsusingdeeplearning'
+            fullPath = os.path.join(self.m_base.m_geoPath, self.m_base.m_mdName)
+            return self.__invokeDynamicFn(
+            [fullPath,
+                    os.path.join(self.m_base.m_geoPath, self.m_base.m_mdName + processKey),
```
Comment (Chamlika) [inline scripts/solutionsLib.py:1419]: Use the variable {fullPath} here

File: scripts/solutionsLib.py [hunk 3]
```diff
@@ -1370,6 +1370,62 @@ def executeCommand(self, com, index=0):
...
+            processKey = 'classifypixelsusingdeeplearning'
+            fullPath = os.path.join(self.m_base.m_geoPath, self.m_base.m_mdName)
+
+            try:
+                out_classified_raster = arcpy.ia.ClassifyPixelsUsingDeepLearning(
+                    fullPath,
+                    os.path.join(self.m_base.const_workspace_path_, 'Parameter/DLPKpackages', self.getProcessInfoValue(processKey, 'in_model_definition', index)),
+                    self.getProcessInfoValue(processKey, 'arguments', index),
+                    self.getProcessInfoValue(processKey, 'processing_mode', index))
+                out_classified_raster.save(os.path.join(self.m_base.m_geoPath, self.m_base.m_mdName + processKey))
```
Comment (Chamlika) [inline scripts/solutionsLib.py:1385]: out_classified_raster.save(f’{fullPath}{processKey}’)

**Review by Chamlika — CHANGES_REQUESTED** [review]
Looks like the changes that you'd done earlier on the PR aren't automatically reflected in the source here. I'd added the requested comments once again, thanks!

**Comment by Chamlika** [comment]
@surabhijain123 This was the earlier PR they'd worked on that have the same code changes and that was approved. Not sure why you'd have multiple issues though on the same changes, perhaps you could discard this issue.

https://github.com/ArcGIS/AID/pull/6829

File: scripts/solutionsLib.py (hunk 1 above)
Comment (surabhijain123) [reply scripts/solutionsLib.py]: Done

File: scripts/solutionsLib.py (hunk 2 above)
Comment (surabhijain123) [reply scripts/solutionsLib.py]: Done

File: scripts/solutionsLib.py (hunk 3 above)
Comment (surabhijain123) [reply scripts/solutionsLib.py]: Done

**Pull request merged**